serializer/seata-serializer-kryo/src/main/java/io/seata/serializer/kryo/KryoSerializer.java

```diff
@@ -27,9 +26,6 @@ public class KryoSerializer implements Serializer {
 
     @Override
     public <T> byte[] serialize(T t) {
-        if (!(t instanceof AbstractMessage)) {
```

Comment:
why delete it ?

Reply:
In order to allow raft to use the kryo serializer, which as it stands has been switched to jackson, it's not needed and I'll make the change right away

Reply:
done